mcs/class/corlib/win32_corlib.dll.exclude.sources

```diff
@@ -0,0 +1,2 @@
+corefx/Interop.GetRandomBytes.cs
+corefx/RandomNumberGeneratorImplementation.Mono.cs
```

Comment:
Are you missing Interop.GetRandomBytes.Mono.cs ?

Reply:
No.